tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -31,12 +31,9 @@ import (
     "github.com/spf13/cobra"
 )
 
-const (
-    PollInterval = time.Second * 2
-    Delay        = time.Second * 5
-)
+const MAX_ACTIVATION_LIMIT = 200
```

Comment:
Is this a CLI limit or a backend limit?  If it's a backend limit, augmenting the existing backend /api/v1 api or add a new one to provide various backend configuration properties/limits would reduce possibility of CLI needing to change in the future if/when this limit is increased.  I suppose this could also be baked into the CLI at compile time, but there's a small exposure of a mismatched CLI/backend.